src/pages/wasmd/getting-started/permissions.mdx

```diff
@@ -0,0 +1,341 @@
+  "deposit": "1000000stake",
+  "title": "Disable Wasm Code Uploads",
+  "summary": "This proposal disables the ability for anyone to upload new WASM binaries.",
+  "expedited": false
+}
+```
+
+**Allow WASM Code Uploads for Everybody**
+
+```json
+{
+  "messages": [
+    {
+      "@type": "/cosmwasm.wasm.v1.MsgUpdateParams",
+      "authority": "wasm10d07y265gmmuvt4z0w9aw880jnsr700js7zslc",
+      "params": {
+        "code_upload_access": {
+          "permission": "Everybody",
+          "addresses": []
+        },
+        "instantiate_default_permission": "Everybody"
+      }
+    }
+  ],
+  "metadata": "ipfs://CID",
+  "deposit": "1000000stake",
+  "title": "Allow WASM code uploads for everybody",
+  "summary": "This proposal enables the ability for anyone to upload new WASM binaries.",
+  "expedited": false
+}
+```
+
+**Restrict WASM Code Uploads to Specific Addresses**
+
+```json
+{
+  "messages": [
+    {
+      "@type": "/cosmwasm.wasm.v1.MsgUpdateParams",
+      "authority": "wasm10d07y265gmmuvt4z0w9aw880jnsr700js7zslc",
+      "params": {
+        "code_upload_access": {
+          "permission": "AnyOfAddresses",
+          "addresses": [
+            "wasm1427kpq9mmnfp0madk5axh2ukmjgpfh6skzdxku",
+            "wasm179avw96aay70ps99umuaesxxczwc0qm5gwufxg"
+          ]
+        },
+        "instantiate_default_permission": "Everybody"
+      }
+    }
+  ],
+  "metadata": "ipfs://CID",
+  "deposit": "1000000stake",
+  "title": "Allow WASM code uploads only for Alice and Bob",
+  "summary": "This proposal enables the ability for Alice and Bob to upload new WASM binaries.",
+  "expedited": false
+}
+```
+
+### Query params
+
+You can query the chain permissions using the following command:
+
+```sh
+wasmd q wasm params -o json
+```
+
+The output looks similar to this:
+
+```json
+{
+  "code_upload_access": {
+    "permission": "Everybody",
+    "addresses": []
+  },
+  "instantiate_default_permission": "Everybody"
+}
+```
+
+## Contract Permissions
+
+This section details how to grant and query permissions for storing, executing, and migrating WASM
+contracts on the blockchain.
+
+### Store Code Authorization
+
+To authorize a specific address to store WASM code, use the following command:
+
+```sh
+wasmd tx wasm grant store-code \
+    wasm1427kpq9mmnfp0madk5axh2ukmjgpfh6skzdxku \
+    3F4CD47C39C57FE1733FB41ED176EEBD9D5C67BAF5DF8A1EEDA1455E758F8514:everybody \
+    --from alice \
+    --keyring-backend=test \
+    --chain-id=docs-chain-1 \
+    -y
+```
+
+This command allows **Alice** to grant **Bob** (`wasm1427kpq9mmnfp0madk5axh2ukmjgpfh6skzdxku`) the
+authorization to **store** a specific WASM code on the blockchain. The WASM code is defined by the
+code checksum `3F4CD47C39C57FE1733FB41ED176EEBD9D5C67BAF5DF8A1EEDA1455E758F8514`. The permission
+granted allows `"everybody"` to instantiate this code. This means that once **Bob** uploads the WASM
+code, any user on the blockchain will be able to instantiate it.
```

Comment:
From this, it's not clear to me what sort of permissions Alice needs to have to be able to do this. Is that important?